VisualizationBase/src/ViewItemManager.cpp

```diff
@@ -121,6 +124,57 @@ void ViewItemManager::removeAllViewItems()
 	currentViewItem_ = nullptr;
 }
 
+ViewItem* ViewItemManager::createIfNotExists(const QString name, QPoint position)
```

Comment:
Alright, I changed the interface a bit. I removed that new method, and made addViewItem() private.  Furthermore, I added a position argument the loadView, such that loadView now doesn't only return the ViewItem, but also adds it to the items internally.